internal/organizations/organization_test.go

```diff
@@ -0,0 +1,176 @@
+	}
+	subCommand := "organization"
+	consumerToken := ""
+	apiURL := ""
+	Version := "metal"
+	rootClient := root.NewClient(consumerToken, apiURL, Version)
+	type fields struct {
+		MainCmd  *cobra.Command
+		Outputer outputPkg.Outputer
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    *cobra.Command
+		cmdFunc func(*testing.T, *cobra.Command)
+	}{
+		{
+			name: "get",
+			fields: fields{
+				MainCmd:  NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				out := &bytes.Buffer{}
+				root.SetArgs([]string{subCommand, "get"})
+				root.SetOutput(out)
+				err := root.Execute()
+				assert.NoError(t, err)
+			},
+		},
+		{
+			name: "create",
+			fields: fields{
+				MainCmd:  NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				out := &bytes.Buffer{}
+				root.SetArgs([]string{subCommand, "create", "-n", "test-org"})
+				root.SetOutput(out)
+				err := root.Execute()
+				assert.NoError(t, err)
+			},
+		},
+		{
+			name: "create",
+			fields: fields{
+				MainCmd:  NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				out := &bytes.Buffer{}
+				root.SetArgs([]string{subCommand, "create", "-n", "test-org1", "-w", "www.metal.equinix.com"})
+				root.SetOutput(out)
+				err := root.Execute()
+				assert.NoError(t, err)
+			},
+		},
+		{
+			name: "create",
+			fields: fields{
+				MainCmd:  NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				out := &bytes.Buffer{}
+				root.SetArgs([]string{subCommand, "create", "-n", "test-org2", "-w", "www.metal.equinix.com", "-t", "https://twitter.com/equinixmetal"})
+				root.SetOutput(out)
+				err := root.Execute()
+				assert.NoError(t, err)
+			},
+		},
+		{
+			name: "update",
+			fields: fields{
+				MainCmd:  NewClient(rootClient, outputPkg.Outputer(&outputPkg.Standard{})).NewCommand(),
+				Outputer: outputPkg.Outputer(&outputPkg.Standard{}),
+			},
+			want: &cobra.Command{},
+			cmdFunc: func(t *testing.T, c *cobra.Command) {
+				t.Helper()
+				root := c.Root()
+				out := &bytes.Buffer{}
+				root.SetArgs([]string{subCommand, "update", "-i", "3bd5bf07-6094-48ad-bd03-d94e8712fdc8", "--name", "test-org2"})
```

Comment:
It would be nice if we could structure these organization tests a little differently so that we could create a test organization, capture the ID of that organization, and then pass that ID in to subsequent update/payment/delete tests.

Reply:
updated